gateway-ha/src/main/java/io/trino/gateway/ha/config/AuthenticationConfiguration.java

```diff
@@ -1,14 +1,91 @@
 package io.trino.gateway.ha.config;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
-@Data
-@AllArgsConstructor
-@NoArgsConstructor
-public class AuthenticationConfiguration {
-  private String defaultType;
-  private OAuthConfiguration oauth;
-  private FormAuthConfiguration form;
+public class AuthenticationConfiguration
+{
+    private String defaultType;
+    private OAuthConfiguration oauth;
+    private FormAuthConfiguration form;
+
+    public AuthenticationConfiguration(String defaultType, OAuthConfiguration oauth, FormAuthConfiguration form)
+    {
+        this.defaultType = defaultType;
+        this.oauth = oauth;
+        this.form = form;
+    }
+
+    public AuthenticationConfiguration() {}
+
+    public String getDefaultType()
+    {return this.defaultType;}
```

Comment:
This format is different from Airlift style. I would recommend formatting (Right click class -> Reformat Code) on IntelliJ after delombok.

Reply:
I know .. all of the codebase does not follow airlift style yet .. the plan was to do a reformat after this PR or ideally after all the current PRs are merged. But given that you started the reformat already I think we should merge this PR .. and then do a reformat of that codebase in a follow up PR same as the other PR you sent.. and then get all other PRs rebased and such